layers/+lang/scala/funcs.el

```diff
@@ -17,6 +17,14 @@
 (defun spacemacs//scala-disable-flycheck-scala ()
   (push 'scala flycheck-disabled-checkers))
 
+(defun spacemacs//scala-backend-ensime ()
+  "Return true if the selected backend is ensime"
```

Comment:
```suggestion
  "Return t if the selected backend is ensime"
```
>In contexts where a truth value is expected, any non-nil value is considered to be true. However, t is the preferred way to represent the truth value true.

https://www.gnu.org/software/emacs/manual/html_node/elisp/nil-and-t.html

Another way to write it seems to be `"Return non-nil ...`, that seems to be the most used out of:
`t`, `true` or `non-nil` in Spacemacs.
These were the number of results when searching with `SPC /` (`HELM Ag: Using rg`)
```
"return.t\b
11

"return.true\b
4

"return.non-nil\b
29
```

The same Emacs manual conventions page also shows the return value as `non-nil`:
>— Function: booleanp object
Return non-nil if object is one of the two canonical boolean values: t or nil.

https://www.gnu.org/software/emacs/manual/html_node/elisp/nil-and-t.html

But it seems like `"Return t ...` probably would be better, because that's what the function returns. And an elisp beginner might expect `"Return non-nil ...` to return `non-nil`.